ras_backstage/controllers/secure_messaging_controller.py

```diff
@@ -13,8 +17,8 @@
 
 def get_messages_list(encoded_jwt, message_args):
     logger.debug('Retrieving messages list', label=message_args.get('label'))
-    url = '{}{}'.format(app.config['RAS_SECURE_MESSAGING_SERVICE'], 'messages')
-    headers = {"Authorization": encoded_jwt}
+    url = '{}{}'.format(app.config['RAS_SECURE_MESSAGING_SERVICE'], 'v2/messages')
```

Comment:
f strings would be more concise.

Reply:
Also all the secure messaging URLs need to be changed to v2 so I'll do the format change separately as part of that PR.